buildSrc/subprojects/profiling/src/main/kotlin/gradlebuild/jmh.gradle.kts

```diff
@@ -13,21 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.gradle.plugins.jsoup
 
-import org.gradle.api.Plugin
-import org.gradle.api.Project
-import org.gradle.api.tasks.Copy
+package gradlebuild
 
-import org.gradle.kotlin.dsl.*
+import me.champeau.gradle.JMHPluginExtension
 
+plugins {
+    id("me.champeau.gradle.jmh")
+}
 
-class JsoupPlugin : Plugin<Project> {
+configurations {
 
-    // TODO Is this used anywhere?
-    override fun apply(project: Project): Unit = project.run {
-        tasks.withType<Copy> {
-            extensions.create<JsoupCopyExtension>("jsoup", this)
-        }
+    getByName("jmhImplementation") {
```

Comment:
:o: could be a bit less wordy:
```kotlin
configurations {
    "jmhImplementation" {
        extendsFrom(configurations["implementation"])
    }
}

Reply:
I've been avoiding change those because `getByName` forces the configuration to exist and the string version will create it.